src/main/resources/graphql/schema.graphqls

```diff
@@ -32,6 +32,7 @@ type Group {
     votingDays: [DayOfWeek]
     votingTime: Time
     isCurrentUserAdmin: Boolean
+    votings(startDate: String!, endDate: String!): [Voting]
```

Comment:
Should use the `DateTime` scalar:

`votings(startDate: DateTime!, endDate: DateTime!): [Voting]`

Reply:
Fixed